types/Email.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const GraphQL = require('graphql');
+
+module.exports = (field) => ({
+	type: GraphQL.GraphQLString,
+	args: {
+		validInput: {
+			type: GraphQL.GraphQLString,
+			description: 'Validates that a valid email has been provided.'
+		},
+	},
+	resolve: (source, args, required, data) => {
+		var value = this.getValueFromData(data);
+		if (value) {
+			return utils.isEmail(value);
```

Comment:
You're returning whether an email is valid here, not the value of the email field.

GraphQL just wants to return the value stored in the field, unless we can think of a requirement that needs more.